l10n_it_account_stamp/models/account_move.py

```diff
@@ -94,11 +104,21 @@ def _move_autocomplete_invoice_lines_values(self):
         return super()._move_autocomplete_invoice_lines_values()
 
     def is_tax_stamp_line_present(self):
+        self.ensure_one()
         for line in self.line_ids:
             if line.is_stamp_line:
                 return True
         return False
 
+    @api.depends(
+        "invoice_line_ids",
+        "invoice_line_ids.product_id",
+        "invoice_line_ids.product_id.is_stamp",
+    )
+    def _compute_tax_stamp_line_present(self):
+        for invoice in self:
+            invoice.tax_stamp_line_present = invoice.is_tax_stamp_line_present()
```

Comment:
`is_tax_stamp_line_present()` non identifica univocamente la riga fattura di imputazione del bollo al cliente perché va a prendere il valore da `is_stamp_line`.

Nel caso di bollo applicato ma non imputato al cliente `tax_stamp_line_present` è sempre `True` e fa comparire "Stamp charged to customer" alla conferma della fattura.
Questo è il comportamento attuale della 16.0 dopo il merge della relativa PR e andrà corretto.

```suggestion
            invoice.tax_stamp_line_present = invoice.is_tax_stamp_product_present()
```